tests/unit/io/test_libevreactor.py

```diff
@@ -72,24 +71,25 @@ def test_watchers_are_finished(self):
 
         @test_category connection
         """
-        with patch.object(LibevConnection._libevloop, "_thread"),\
-             patch.object(LibevConnection._libevloop, "notify"):
+        from cassandra.io.libevreactor import _global_loop
```

Comment:
Seems like the import has to happen here so the it happens after `_global_loop` is initialized, if it's imported at the beginning, `_global_loop` is `None`